README.md

```diff
@@ -61,15 +61,26 @@ CONTENTFUL_SPACE_ID=12345_Secret_SpaceId
 The following data is required to configure the storefront to communicate with your Kibo API Client.
 
 - `apiHost` - Your Kibo Commerce API Host.
-- `authHost` - Kibo Commerce Authentication Host Server. It is used to request an access token from Kibo Commerce OAuth 2.0 service. Production and Production sandbox, use `home.mozu.com`
-- `clientId` - Unique Application (Client) ID of your Application
-- `sharedSecret` - Secret API key used to authenticate application. Viewable from your [Kibo eCommerce Dev Center](https://mozu.com/login)
-- `contentful-access-token` - Unique token used to authenticate your contentful cms
-- `Contentful-preview-mode` - Unique token used to authenticate preview mode for contentful
-- `Contentful-space-id`- Unqiue space id to identify contentful application space
+- `authHost` - Kibo Commerce Authentication Host Server. It is used to request an access token from Kibo Commerce OAuth 2.0 service. Production and Production sandbox, use `home.mozu.com`.
+- `clientId` - Unique Application (Client) ID of your Application.
+- `sharedSecret` - Secret API key used to authenticate application. Viewable from your [Kibo eCommerce Dev Center](https://mozu.com/login).
 
 Visit [Kibo documentation](https://apidocs.kibong-perf.com/?spec=graphql#auth) for more details on API authentication
 
+next.config.js example
+
+```bash
+contentFulAccessToken= 123_access_token
+contentFulPreviewToken= 123_preview_token
```

Comment:
whenever you're adding examples to docs, use the same format. "Next.config.js" is a javascript file
Dont use bash lines.

I'm not sure what these values mean. They don't map to environment variables or anything in the config

Add the relevant snippet from the next config here in the docs
and, in one line, explain what/why they're here. 
Mention,

```javascript
// next.config.js
{
  //...other config
   contentful: {
      accessToken: process.env.CONTENTFUL_ACCESS_TOKEN,
      previewAccessToken: process.env.CONTENTFUL_PREVIEW_ACCESS_TOKEN,
      URL: `https://graphql.contentful.com/content/v1/spaces/${process.env.CONTENTFUL_SPACE_ID}/environments/master`,
    }
  }
```

Reply:
Resolved , updated readme as mentioned